src/kili/services/plugins/model.py

```diff
@@ -105,8 +105,21 @@ def on_custom_interface_click(
         so it should not be used.
 
         Args:
-            label: label submitted to Kili
+            label: Label submitted to Kili: a dictionary containing the following fields:
+                `id`, `labelType`, `numberOfAnnotations`, `authorId`, `modelName`, `jsonResponse`,
+                `secondsToLabel`, `isSentBackToQueue`, `search` and some technical fields:
+                `createdAt`, `updatedAt`, `version`, `isLatestReviewLabelForUser`,
+                `isLatestLabelForUser`, `isLatestDefaultLabelForUser`, `readPermissionsFromProject`.
             asset_id: id of the asset on which the action is called
+
+        Example use:
+
+            >>> def on_custom_interface_click(self, label: Label, asset_id: str):
+            >>>     json_response = label.get('jsonResponse')
+            >>>     if label_is_respecting_business_rule(json_response):
+            >>>         return
+            >>>     else:
+            >>>         self.kili.send_back_to_queue(asset_ids=[asset_id])
```

Comment:
No `send_back` in this example, since it should be triggered by the labeler himself. A `create_issues` would be more appropriate

Reply:
I agree, it feel weird to have the same example over and over again, and it doesn't fit this handler. Thanks for the insight